app/imports/api/utilities/transactionUtils.js

```diff
@@ -40,7 +40,7 @@ export function determineNextPayment(tPayPeriod, currentD) {
 }
 
 export function saveTransactionForRecord(transactionArray, currentDate, action) {
-  transactionArray.push({ date: moment(currentDate).format('DD Mo YYYY'), action: action });
+  transactionArray.push({ date: moment(currentDate).toISOString(), action: action });
```

Comment:
@htobin , this is what i meant with iso string. see examples in the transaction modal default logs.